hmda/src/main/scala/hmda/api/http/admin/InstitutionAdminHttpApi.scala

```diff
@@ -156,6 +166,29 @@ trait InstitutionAdminHttpApi extends HmdaTimeDirectives {
     }
   }
 
+  private def validTaxIdFormat(taxIdOption: Option[String]): Boolean = {
```

Comment:
You should be able to simplify this to a regex 

Example:

val taxIDPattern ="[0-9]{2}\\-[0-9]{7}$".r

taxID ="12-1234567"

taxIDPattern.matches(taxID)  // returns true